web/src/hooks/use-market-item.hook.js

```diff
@@ -3,12 +3,17 @@ import {sansPrefix} from "@onflow/fcl"
 import {IDLE, PROCESSING} from "../global/constants"
 import {useCurrentUser} from "../hooks/use-current-user.hook"
 import {useAccountItems} from "../hooks/use-account-items.hook"
-import {useMarketItems} from "../hooks/use-market-items.hook"
 import {useKibblesBalance} from "../hooks/use-kibbles-balance.hook"
 import {fetchMarketItem} from "../flow/fetch-market-item.script"
 import {buyMarketItem} from "../flow/buy-market-item.tx"
 import {cancelMarketListing} from "../flow/cancel-market-listing.tx"
 
+class SaleOffer {
+  constructor(accountItem) {
+    if(!accountItem)
+  }
+}
```

Comment:
How does this work? I'm curious about the `if(!accountItem)` line -- is that shorthand?

Reply:
This is incomplete. 😅

Reply:
Not going to use a class for this, just going to create a normalizing function. Something that converts from snake case (result from the db query) to camel case.